core/src/main/java/nl/nn/adapterframework/doc/model/FrankElement.java

```diff
@@ -177,10 +224,57 @@ public void handleCumulativeChildrenOf(FrankElement frankElement) {
 		return result;
 	}
 
+	/**
+	 * Get the XSD element name, which depends on the {@link ElementRole}. For
+	 * example, a <code>nl.nn.adapterframework.senders.CommandSender</code>
+	 * has tag <code>CommandErrorSender</code> when used as an error sender
+	 * and tag <code>CommandSender</code> when used as a sender.
+	 * <p>
+	 * When the {@link ElementType} is not interface-based, we have to do with
+	 * elements like <code>&lt;Param&gt;</code> or <code>&lt;Forward&gt;</code>.
+	 * These names are derived from the syntax 1 name of the {@link ElementRole}.
+	 * <p>
+	 * The above rules are not sufficient however to produce the
+	 * <code>compatibility.xsd</code> of the XSD. In that case, there are
+	 * multiple {@link ElementRole} objects owning the same {@link FrankElement}.
+	 * We have to filter out {@link ElementRole} objects coming from deprecated
+	 * config children. This is achieved using the <code>owningElementRole</code>
+	 * field.
```

Comment:
Dat is toch geen probleem?
Wat er lastig is, is dat bepaalde ElementRoles een syntax1naam delen, maar dat is alleen maar lastig omdat je het syntax1-style element maar één keer op moet nemen.